src/github/client.js

```diff
@@ -133,6 +134,36 @@ function GithubClient(url, username, privateToken) {
 			});
 	};
 
+	this.createWebhook = function (repoName, secret, events, payloadUrl, orgName) {
+		var path = `repos/${orgName}/${repoName}/hooks`;
+		var data = {
+			'events': events,
+			'config': {
+				'url': payloadUrl,
+				'content_type': 'json',
+				'insecure_ssl': '0',
+				'secret': secret
+			}
+		};
+
+		var params = this._getParams('POST', path);
+		params.data = data;
+		return axios(params)
+			.then(response => {
+				console.info('Created webhook for %s', repoName);
+				return response;
+			})
+			.catch((error) => {
+				console.error(error);
+				if (error.response.status === 422) {
+					throw new WebhookError(error.response.status, `Webhook already exists for repo ${repoName}`);
```

Comment:
Do we really need WebhookError to be thrown from here?
Essentially we just need to make the call `idempotent`, so let's just warn and absorb the error.

I have tried doing similar in createRepo.

Reply:
yes we do need to throw because the user interfacing component should be responsible for handling it gracefully. Otherwise, consuming code won't be able to differentiate between genuine error and when webhook is already created

Reply:
We need to define criteria for idempotent for webhooks. There can be multiple webhooks for a repository. As discussed, we can remove this check all together for 422 status check.